source/localizable/feedspec/index.en.html.md.erb

```diff
@@ -594,6 +596,225 @@ For example: If there are 10 available pieces of Medium size, 15 of Large and 5
+<%= settings.site_name.capitalize %>:
+
+<div class="alert alert-danger">
+  <p>Currently, size variations can only be used in products in which size is a required field.</p>
+</div>
+
+<div class="alert alert-danger">
+  <p>
+    You cannot include HTML notation in <strong>any</strong> of the size variation's
+    attributes.
+  </p>
+</div>
+
+Name                                                   | Type    | Length                                               | Required | Beta<b><sup>1</sup></b> | Example
+------------------------------------------------------ | ------- |------------------------------------------------------|----------| --------------------------------------------
+[Variation Unique ID](#variation-unique-id)            | String  | <span class="mono">200</span>                        | Yes      |  No                     | `12431.XL`
+[Variation Availability](#variation-availability)      | String  | <span class="mono">100</span>                        | Yes      |  No                     | `In 2 days`
+[Variation Size](#variation-size)                      | String  | <span class="mono">64</span>                         | Yes      |  No                     | `M`
+[Variation Quantity](#variation-quantity)              | Integer | <span class="mono">8</span>                          | Yes      |  No                     | `10`
+[Variation Link](#variation-link)                      | String  | <span class="mono">2000</span>                       | No       |  Yes                    | `https://www.someurl.co.uk/products/322.html`
+[Variation Price](#variation-price)                    | Decimal | <span class="mono">- (up to 2 decimal places)</span> | No       |  Yes                    | `23.22`
+[Variation MPN](#variation-mpn)                        | String  | <span class="mono">80</span>                         | No       |  Yes                    | `GF322234`
+[Variation EAN / Barcode](#variation-ean--barcode)     | Integer | <span class="mono">80</span>                         | No       |  Yes                    | `9780471117094`
+
+<small><b><sup>1</sup></b> Beta fields are recommended to be included in the XML. However, their values
+will not be used initially and will be overridden by their respective product attributes. Also these
+fields are subject to removal from the XML feed specification.</small>
+
+#### Variation Unique ID
+
+This attribute represents the **unique** identifier of a size variation across
+shop in your database. **The value of this field must remain the same through
+a size variation's lifecycle and cannot be reused in other variations**.
+
+> A size Variation Unique ID cannot refer to **more than** one size variation. In such a
+case the size variation will be updated only the first time it's encountered in the
+data feed.
+
+> In case your e-commerce application supports more than one languages your
+data feed should only include definitions in the English language to avoid
+duplication.
```

Comment:
```suggestion
> In case your e-commerce application supports more than one languages, your
data feed should only include definitions in the English language to avoid
duplication.
```